examples/nextjs-comments-video/src/components/Threads.module.css

```diff
@@ -0,0 +1,56 @@
+.emptyState {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100%;
+  color: var(--gray9);
+  padding: var(--spacing);
+}
+
+.threadWrapper {
+  position: relative;
+  border-bottom: 1px solid var(--gray4);
+  transition: background 0.4s ease;
+}
+
+.threadWrapper[data-highlight] {
+  background: var(--gray3);
+  transition: background 0.1s ease;
+}
+
+.thread {
+  --lb-background: none;
```

Comment:
Doing this can cause visual issues, could we instead change it to the value you want on hover?